tests/DependencyInjection/SonataMediaExtensionTest.php

```diff
@@ -224,6 +226,7 @@ public function dataFilesystemConfiguration(): array
                             'sdk_version' => 3,
                             'region' => 'region',
                             'version' => 'version',
+                            'endpoint' => null,
```

Comment:
I think this line should be removed, people not using endpoint will not define it to null, they will simply not have it defined at all